### ISSUE TITLE
topic is removed from message?

### ISSUE THREAD

**Issue by Fernando-Checa** [mochman/node-red-contrib-light-transition#23]
### Terms

- [X] This request is not a duplicate of an existing issue.
- [X] I have provided as much information as I can to help resolve this issue.

### Description

It seems the node removes TOPIC from the message.
Is there a way to keep the topic through the node?

### steps-to-reproduce

Feed a message with topic and it comes out without

### Actual Behavior

Feed a message with topic and it comes out without

### Version

1.5.1

### JSON of node setup

```shell
[
    {
        "id": "5f156cada4cfd3a0",
        "type": "Thing Update",
        "z": "39f9e2f8e099f89d",
        "nodeName": "update light State",
        "name": "",
        "thingType": "Fights",
        "debug": false,
        "updates": [],
        "x": 670,
        "y": 3240,
        "wires": []
    }
]
```

**Comment by mochman** [comment]
Sure thing.  I just released v1.6.0 that now passes through the original msg object (excluding msg.payload).  msg.topic and any other msg.SOMETHING should come through on all the outputs.  Please let me know if that works like you wanted it to.

**Comment by Fernando-Checa** [comment]
Just tried the new version and works like a charm.

Thank you!
